data/src/entity_manager/system.py

```
@@ -7176,7 +7203,7 @@ def normalize_options(self, options):
         # the options map (for lazy loading) only in case the
         # options is a dictionary
         is_normalized = (
-            type(options) == dict and options.get("_normalized", False) or False
+            options.get("_normalized", False) if type(options) == dict else False
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)